common/models/templates/src/tokenization.ts

```diff
@@ -1,129 +1,129 @@
+/// <reference types="@keymanapp/models-types" />
```

Comment:
A bunch of changes in this file don't immediately appear to be related to the change to ESModules?

Reply:
I remember that during some of the modularization work, VS Code's TS engine got a _lot_ stricter about types for a while.  (It particularly didn't like certain things that were happening when `undefined` was possible.)  

I'm not sure if this was prompted by conversion to modules or what, but I figured small changes to satisfy the stricter type checks couldn't hurt.